SOOP/SOOP_ASF_SST_XBT/SOOP_BOM_ASF_SST.py

```diff
@@ -68,15 +67,31 @@ def parse_arg():
     parser = argparse.ArgumentParser()
     parser.add_argument("-f", "--force-push-incoming",
                         help="force the push of all files alreay downloaded to the incoming dir. Does not download new ones", action="store_true")
+    parser.add_argument("-r", "--reprocess-regexp", type=str, help="reprocess all files already downloaded matching a regexp. '*SOOP-SST*' '*FHZI*' ... to the incoming dir", )
```

Comment:
According to the docs for `fnmatch`, it's not a regular expression match but a much simpler shell pattern match.